providers/PrivyProvider.tsx

```diff
@@ -15,6 +15,9 @@ const PrivyProvider = ({ children }: PrivyProviderProps) => (
         accentColor: '#676FFF',
         logo: '/images/logo2.png',
       },
+      embeddedWallets: {
+        createOnLogin: 'all-users',
```

Comment:
Let's set this to only users without wallets.
- moving forward, we could add new `wallet` logins to privy. In that case, we would not want to create a new, embedded, wallet right???
What do you think???